frontend/spec/controllers/spree/orders_controller_spec.rb

```diff
@@ -20,24 +21,46 @@
       end
 
       context "with Variant" do
-        let(:populator) { double('OrderPopulator') }
-        before do
-          expect(Spree::OrderPopulator).to receive(:new).and_return(populator)
-        end
-
         it "should handle population" do
-          expect(populator).to receive(:populate).with("2", "5", nil).and_return(true)
-          spree_post :populate, { :order_id => 1, :variant_id => 2, :quantity => 5 }
+          expect {
+            spree_post :populate, { :variant_id => variant.id, :quantity => 5 }
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.